src/main/webapp/app/complaints/list-of-complaints/list-of-complaints.component.ts

```diff
@@ -20,15 +22,16 @@ import { faExclamationTriangle, faFolderOpen, faSort } from '@fortawesome/free-s
     providers: [],
 })
 export class ListOfComplaintsComponent implements OnInit {
+    readonly ComplaintType = ComplaintType;
+
     public complaints: Complaint[] = [];
-    public hasStudentInformation = false;
     public complaintType: ComplaintType;
-    ComplaintType = ComplaintType;
 
     private courseId: number;
     private exerciseId: number;
     private tutorId: number;
     private examId?: number;
+    course?: Course;
```

Comment:
Do we still need the courseId? I think we can simply use course.id instead

Reply:
Yes, it gets used above line 105 call to create an Observable for the complaints. This allows us to run both requests in parallel (we already have the ID so why wait for the whole course object to then read the id?).